charts/loki-simple-scalable/templates/read/statefulset-read.yaml

```diff
@@ -74,8 +76,6 @@ spec:
           volumeMounts:
             - name: config
               mountPath: /etc/loki/config
-            - name: tmp
```

Comment:
why are we removing the `/tmp` volume? I'm not actually sure what it's used for, but I can't tell from this PR why it's being removed either.